general/marketing_and_branding.rst

```diff
@@ -0,0 +1,193 @@
+This section hopes to provide some basic information how how ACM does various
+marketing and branding items like making fliers, ordering marketing equipment,
+branding rules, etc.
+
+.. _general-marketing_and_branding-resources:
+
+Marketing Approaches + Resources
+---------------------------------
+1. **CSTask** - You can send your flyers or advertisements to cstask (email
+   maintained by the CS Office staff). They will print the number of flyer
+   copies you request and can also send the flyer or advertisements to all
+   computer science students. Ask the current President or the CS Office Staff
+   for the email address for cstask.
+2. **ACM Google Groups** - ACM maintains multiple google groups with student
+   emails depending on the different perferences. You should send any
+   advertisements or flyers to the pertaint list. Ask the current President for
+   more information.
+3. **Hanging Flyers** - Various buildings will allow you to hang flyers or
+   advertisements up. See the :ref:`general-marketing_and_branding-locations`
+   section for more information.
+4. **Word of Mouth** - Definitely the most powerful form of advertisement.
+   Send out personal invitations to people you think who would be interested in
+   coming or have people themselves spread the word. Most people will ignore
+   emails or flyers but are much more likely to come when directly invited
+
+
+Flyers
+-------
+The majority of the marketing efforts for ACM is through flyers. Whenenver ACM
+hosts an event, an effort should be made to design, develop, and post flyers
+around the pertinent school buildings so that students can be made.
+
+.. note::
+   Certain buildings will only allow you to hang fliers if there is the
+   Missouri S&T logo somewhere on that flier. Moreover, before you hang up a
+   flyer in any building, be sure to ask the front office for permission.
+
+.. _general-marketing_and_branding-locations:
+
+Hanging Locations
+^^^^^^^^^^^^^^^^^
+Please see the table below for the buildings we have hung flyers in, the number
+of flyers necessary for the building, and if you need approval to hang items in
+the building.
+
+If you need tape to hang the flyers, ask the CS Office.
+
+.. note::
+   Most buildings have designated boards where students must hang flyers. Please
+   do not deviate from these areas as they may no longer allow students to hang
+   flyers.
+
+==========================  ================   ==================
+Location                    Number of Flyers   Approval Required?
+==========================  ================   ==================
+Computer Science (CS)       25                 Ask CS 327
+Thomas Jefferson (TJ)       **TBD**            Ask front desk, they will hang
+Emerson Electric (EE)       3                  Ask office supervisor
+Residental Commons 1 (RC1)  **TBD**            Ask front desk
+Residental Commons 2 (RC2)  **TBD**            Ask front desk
+University Commons (UC)     **TBD**            Ask front desk
```

Comment:
Still waiting on the number for this one, so don't merge this until I get a return